apache/zeppelin
# [ZEPPELIN-2382] Fixed windows compatibility of interpreter binding.

### What is this PR for?
Fixing ZEPPELIN-2382:

A fresh install of Zeppelin 0.7.1 failed to start on a Windows due to the use of [getPosixFilePermissions](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#getPosixFilePermissions(java.nio.file.Path,%20java.nio.file.LinkOption...))

This method fails on Windows with an UnsupportedOperationException - however, recovering (that is ignoring the exception) appears to have no negative effects. (For a discussion on this related to a different project see, e.g., https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/937 )

### What type of PR is it?
[Bug Fix]

### Todos
* [x] - Have the binary distribution checked by the reporter of [ZEPPELIN-2382]

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-2382

### How should this be tested?
Check if clean distribution runs on Windows. Important: Ensure that HADOOP_HOME points to a folder containing bin/winutils.exe since this pull request does not include a fix for https://issues.apache.org/jira/browse/ZEPPELIN-2438 (which is part of a separate pull request).

### Screenshots (if appropriate)
N/A

### Questions:
* Does the licenses files need update?
No

* Is there breaking changes for older versions?
No

* Does this needs documentation?
No
